gh_actions_exporter/config.py

```diff
@@ -17,33 +16,19 @@ def yaml_config_settings_source(settings: BaseSettings) -> Dict[str, Any]:
     return {}
 
 
-class RelabelType(str, Enum):
-    name = 'name'
-    label = 'label'
-
-
-class Relabel(BaseModel):
-    label: str
-    type: RelabelType = RelabelType.label
-    description: Optional[str]
-    default: Optional[str] = None
-    values: List[str]
-
-
```

Comment:
While this will not be used for costs, it is being used to be able to label other type of prometheus metrics, you can't delete this feature.